README-zh-cn.md

```
@@ -0,0 +1,276 @@
+   - Name: `zerodotemail`
+   - Username: `postgres`
+   - Password: `postgres`
+   - Port: `5432`
+
+2. **设置数据库连接**
+
+   确保你的数据库连接字符在 `.env`文件中.
+
+   对于本地开发用户:
+
+   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)